l10n_nl_xaf_auditfile_export/tests/test_l10n_nl_xaf_auditfile_export.py

```diff
@@ -28,16 +28,18 @@ def setUp(self):
     def test_l10n_nl_xaf_auditfile_export_default(self):
         self.export.button_generate()
         self.assertTrue(self.export.auditfile)
-        zf = StringIO(b64decode(self.export.auditfile))
+        zf = BytesIO(b64decode(self.export.auditfile))
         with ZipFile(zf, 'r') as archive:
-            contents = archive.read(archive.namelist()[0])
-        self.assertTrue(contents.startswith('<?xml '))
+            filelist = archive.filelist
+            contents = archive.read(filelist[-1]).decode('utf8')
+        self.assertEquals(contents[:6], '<?xml ')
 
     def test_l10n_nl_xaf_auditfile_export_all(self):
         self.export.write({'data_export': 'all'})
         self.export.button_generate()
         self.assertTrue(self.export.auditfile)
-        zf = StringIO(b64decode(self.export.auditfile))
+        zf = BytesIO(b64decode(self.export.auditfile))
         with ZipFile(zf, 'r') as archive:
-            contents = archive.read(archive.namelist()[0])
-        self.assertTrue(contents.startswith('<?xml '))
+            filelist = archive.filelist
+            contents = archive.read(filelist[-1]).decode('utf8')
```

Comment:
Also now in line with [10.0](https://github.com/OCA/l10n-netherlands/blob/10.0/l10n_nl_xaf_auditfile_export/tests/test_l10n_nl_xaf_auditfile_export.py#L42).